src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -467,6 +469,14 @@ public FeedValidationResultSummary latestValidation() {
         return resourceTable;
     }
 
+    /**
+     * Find all project feed sources that contain the label and remove label from list.
+     */
+    public static void removeLabelFromFeedSources(Label label) {
+        Bson query = and(eq("projectId", label.projectId), in("labelIds", label.id));
+        Persistence.feedSources.updateMany(query, pull("labelIds", label.id));
+    }
+
```

Comment:
I think `noteCount()` may be doing something funky, as the number of notes changes when I click on the tab (when logged in as a non-admin user)

https://user-images.githubusercontent.com/86619099/128217174-a5f7f903-810b-4409-a144-374969e4a593.mov

Reply:
Wow, nice catch. Updated in 778b8b68f80bc82c572a75c422297abd87153b81. Specifically here: https://github.com/ibi-group/datatools-server/commit/778b8b68f80bc82c572a75c422297abd87153b81#diff-32b0511be9a360c3dc29348a279d1695ad488c91ab04d7d7fcf44b5103b03843R387-R390 (because that ended up being a big commit).